Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -193,6 +144,59 @@ public function subgraphForNode(Node $node): ContentSubgraphInterface
         return $this->contentRepositoryRegistry->subgraphForNode($node);
     }
 
+    /**
+     * Check if the given node is already a collection, find collection by nodePath otherwise, throw exception
+     * if no content collection could be found
+     *
+     * @throws Exception
+     * @internal implementation detail of Neos.Neos:ContentCollection
+     */
+    public function nearestContentCollection(Node $node, ?string $nodePath): Node
```

Comment:
Why don't we just forbid `null` nodePaths here. Also Sneakily added internal, that's no good as this exists since ages and ever was internal.

Reply:
the helper is just a bit weird and mostly (see docs) only exists to improve errors if no content collection could be found.

The main change of this pr was to make it nullable again as otherwise a php type error will be raised instead of a human readable 

> No content collection of type %s could be found in the current node and no node path was provided.'
>                     . ' You might want to configure the nodePath property'
>                     . ' with a relative path to the content collection

The helper doesnt do anything that is not doable via native eel and fusion, that is why i added the `@internal` annotation as it very much looks like just to be a means to an end of the `Neos.Neos:ContentCollection`.
It just seems like a hack an with the `Neos.Node` utility now becoming more important to access node things i rather like to hide the ugly parts:)

Reply:
Mmm, yes works for me, lets go with it.